.github/workflows/ruby.yml

```diff
@@ -29,6 +32,7 @@ jobs:
         run: |
           sudo apt-get -yqq install libpq-dev
           gem install bundler
+          cd src
```

Comment:
we can use instead? also in the `yarn` step?
```
working-directory: ./src
```

Reply:
where should I write this line ?